### PR TITLE
Update dependencies to Citadel

### PR DESCRIPTION
**Pull request by j-rivero**

### CODE REVIEW

File: ros_ign_image/package.xml [hunk 1]
```diff
@@ -7,8 +7,8 @@
 
   <buildtool_depend>catkin</buildtool_depend>
 
-  <depend>ignition-msgs4</depend>
-  <depend>ignition-transport7</depend>
```
Comment (chapulina) [inline ros_ign_image/package.xml]: `ros_ign_bridge` also needs a `package.xml` update

File: ros_ign_image/CMakeLists.txt [hunk 2]
```diff
@@ -16,11 +16,22 @@ find_package(catkin REQUIRED COMPONENTS
   roscpp
```
Comment (chapulina) [inline ros_ign_image/CMakeLists.txt]: Note that `ros_ign_image` is currently [being ignored](https://github.com/osrf/ros_ign/blob/dashing/ros_ign_image/COLCON_IGNORE) for ROS 2. But I think it doesn't hurt to update the cmake to match melodic's.

File: ros_ign_image/package.xml (hunk 1 above)
Comment (j-rivero) [reply ros_ign_image/package.xml]: I think that we have now them updated after the merge of https://github.com/osrf/ros_ign/pull/58

File: ros_ign_image/CMakeLists.txt [hunk 3]
```diff
@@ -48,6 +48,9 @@ target_link_libraries(${executable}
   ignition-msgs${IGN_MSGS_VER}::core
   ignition-transport${IGN_TRANSPORT_VER}::core
 )
+install(TARGETS ${executable}
+        DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION}
+)
```
Comment (chapulina) [inline ros_ign_image/CMakeLists.txt]: This is already below on line 62, and has been updated for the `ament` pattern

File: ros_ign_bridge/package.xml [hunk 4]
```diff
@@ -12,6 +12,8 @@
 
   <buildtool_depend>ament_cmake</buildtool_depend>
 
+  <depend>ignition-msgs5</depend>
+  <depend>ignition-transport8</depend>
```
Comment (chapulina) [inline ros_ign_bridge/package.xml:16]: Nice catch :+1:

**Review by chapulina — CHANGES_REQUESTED** [review]

**Review by chapulina — APPROVED** [review]

**Pull request merged**